txp/Pos/Txp/Toil/Failure.hs

```diff
@@ -55,5 +62,17 @@ instance Buildable ToilVerFailure where
     build (ToilTooLargeTx {..}) =
         bprint ("transaction's size exceeds limit "%
                 "("%memory%" > "%memory%")") ttltSize ttltLimit
+    build (ToilInvalidMinFee {..}) =
+        bprint ("policy"%build%" generates invalid minimal fee on a "%
```

Comment:
1. You forgot space.
2. Word `policy` seems to be redundant, because format of `TxFeePolicy` already includes it.